Java/src/main/java/com/gildedrose/DefaultQualityUpdater.java

```diff
@@ -0,0 +1,16 @@
+package com.gildedrose;
+
+public class DefaultQualityUpdater extends QualityUpdater {
+    @Override
+    public boolean handlesItem(Item item) {
+        return true;
+    }
+    @Override
+    public void updateQuality(Item item){
+        decreaseQuality(item);
```

Comment:
This encapsulates the "business logic" of an item that is losing quality very well.